_std/__build.dm

```diff
@@ -117,38 +117,40 @@ o+`        `-` ``..-:yooos-..----------..`
 //#define USE_PERSPECTIVE_EDITOR_WALLS
 
 //------------- MAP OVERRIDES -------------//
+
+//#define MAP_OVERRIDE_DEVTEST			// Developer Testing map, by cringe
+
 //-------Special Events:
 //#define MAP_OVERRIDE_CONSTRUCTION	// Construction mode
 //#define MAP_OVERRIDE_POD_WARS   	// 500x500 Pod Wars map
 //#define MAP_OVERRIDE_EVENT      	// Misc. event maps
-//#define MAP_OVERRIDE_PAMGOC				// Pamgoc, cogmaP
-//#define MAP_OVERRIDE_WRESTLEMAP   // Wrestlemap by Overtone
+//#define MAP_OVERRIDE_PAMGOC				// ilaP yB ,1 pamgoC (pamgoc by pali)
+//#define MAP_OVERRIDE_WRESTLEMAP   // Wrestlemap, by Overtone
 
 //-------Rotation maps:
 //#define MAP_OVERRIDE_COGMAP				// Cogmap1, by Dr. Cogwerks
 //#define MAP_OVERRIDE_COGMAP2			// Cogmap2, by Dr. Cogwerks
-//#define MAP_OVERRIDE_DONUT2				// Updated Donut2, by committee. Currently managed by Sord.
-//#define MAP_OVERRIDE_DONUT3				// Donut3 by Ryumi
-//#define MAP_OVERRIDE_KONDARU			// Kondaru by Kubius
-//#define MAP_OVERRIDE_ATLAS				// Atlas by Gannets (and Kubius)
-//#define MAP_OVERRIDE_CLARION			// Used to be Destiny's Alt. By Dionsu and a69andahalf.
-//#define MAP_OVERRIDE_OSHAN				// Oshan by committee
+//#define MAP_OVERRIDE_DONUT2				// Updated Donut Station 2, by committee.
+//#define MAP_OVERRIDE_DONUT3				// Donut Station 3, by Ryumi
+//#define MAP_OVERRIDE_KONDARU			// Kondaru Station, by Kubius
+//#define MAP_OVERRIDE_ATLAS				// NSS Atlas, by Gannets (and Kubius)
```

Comment:
NCS Atlas. Not all ship maps use the NSS prefix. Please check all the map names in the relevant map config datums to make sure you're getting them right.